clusters/ztp-siteconfig/bevo2.cars2.lab/bevo2-siteconfig.yaml

```diff
@@ -0,0 +1,92 @@
+---
+# yamllint disable rule:line-length
+# yamllint disable rule:comments-indentation
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "bevo2"
+  namespace: "bevo2"
+spec:
+  baseDomain: "cars2.lab"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "img4.14.10-x86-64-appsub"
+  sshPublicKey: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDd7Jj5iFCWv9IHJK9H+2O3lyPs36moAxeAUiHvzRS3uzqGxxB33BnTRBNDKsoDFSGJX0J4bd5b+XyCPdhFOfvn/xhmAcm6d8GALS+139e8d+No8h2QgZy0OVJFp844k4nmz4wew5/+X9DN40ZURYerekbVc58hw1+rTu0uM2jQ0cE2QmEf3qGKHx9UJW8t6IsMzwnrikBH30sYqn2NcBE+/c8JzlLc3PvvenlY0iQkpukI1A5E9GGMR9OS/q+w6FH85zvSgUatOV7Q5lg45QUF+V77DrfX5+niI+NK1g70pRvD8481SAdXrHPB5vK4vQEmJ4pz83IKYHVuPzRnjzYKv1jV33oReyyMqyk44Rsfkxl4i5SJ9z7q/EVmTjvurzD6ofi3Dg0+PL18eTcjuPFdCxSCUFsnr5N9CRHCxHRQpxoZTD7sYD4jDGNygawLvhxcvgKGBZzP53NRCzRFOMFmZsLPLQRaNOsgKRPAohmrn5l8+1xG5ltVauOwAFlKUxk="
+  clusters:
+    - clusterName: "bevo2"
+      networkType: "OVNKubernetes"
+      installConfigOverrides: "{\"capabilities\":{\"baselineCapabilitySet\": \"None\", \"additionalEnabledCapabilities\": [ \"marketplace\", \"NodeTuning\" ] }}"
+      cpuPartitioningMode: AllNodes
+      clusterLabels:
+        common-du-414: true
+        group-dellr750-vse6: ""
+      clusterNetwork:
+        - cidr: 10.128.0.0/14
+          hostPrefix: 23
+        - cidr: fd01::/48
+          hostPrefix: 64
+      machineNetwork:
+        - cidr: 192.168.38.128/26
+        - cidr: 2600:52:7:300::0/64
+      serviceNetwork:
+        - 172.30.0.0/16
+        - fd02::/112
+      additionalNTPSources:
+        - clock.cars2.lab
+        - clock-v6.cars2.lab
+      # Optionally; This can be used to override the KlusterletAddonConfig that is created for this cluster:
+      # crTemplates:
+      # KlusterletAddonConfig: "KlusterletAddonConfigOverride.yaml"
+      # proxy:
+      # httpProxy: http://cars2-client.infra.cars2.lab:3128
+      # httpsProxy: http://cars2-client.infra.cars2.lab:3128
+      # noProxy: ".cars2.lab,2600:52:7:300::0/64,fd02::/112,fd01::/48,2600:52:7:300::177,2600:52:7:300::179,2600:52:7:300::180,2600:52:7:300::181"
+      nodes:
+        - hostName: "r750-2.bevo2.cars2.lab"
+          role: "master"
+          # Optionally; This can be used to configure desired BIOS setting on a host:
+          # biosConfigRef:
+          # filePath: "example-hw.profile"
+          bmcAddress: "idrac-virtualmedia://192.168.38.204/redfish/v1/Systems/System.Embedded.1"
+          bmcCredentialsName:
+            name: "r750-2-bmc-creds-secret"
+          bootMACAddress: "b4:96:91:a5:04:ac"
+          # Use UEFISecureBoot to enable secure boot
+          bootMode: "UEFI"
+          rootDeviceHints:
+            deviceName: "/dev/disk/by-path/pci-0000:65:00.0-scsi-0:2:0:0"
+          nodeNetwork:
+            interfaces:
+              - name: ens2f0
+                macAddress: "b4:96:91:a5:04:ac"
```

Comment:
```suggestion
          nodeNetwork:
            interfaces:
              - name: eno12399np0
                macAddress: "e4:3d:1a:dd:b8:20"
              - name: ens2f0
                macAddress: "b4:96:91:a5:04:ac"
              - name: ens5f0
                macAddress: "50:7c:6f:1f:b3:58"
```

Reply:
add broadcom interface like bevo1.  also add other connected (with carrier) interfaces.

Reply:
Added a few more, and ipaddress confiuration in the broadcom.